packages/docs-reanimated/docs/layout-animations/keyframe-animations.mdx

```diff
@@ -96,117 +25,68 @@ const keyframe = new Keyframe({
     transform: [{ rotate: '45deg' }],
   },
 });
+
+function App() {
+  return <Animated.View entering={keyframe} />;
+}
 ```
 
-## How to use keyframe animations?
+### Arguments
 
-Currently, you can define animations using keyframes only for entry and exit animations.
+#### `definitions`
 
-### 1. Choose Animated Component which entering or exiting you want to animate
+An object, passed as a first argument that contains definitions of your animation.
```

Comment:
If we add the `Type definitions` section, then there is no need to specifiy it is a first argument - people will see it clearly 😄 
```suggestion
An object, that contains definitions of your animation.
```